update-fake.bat

```diff
@@ -0,0 +1,3 @@
+cls
+Packages\NuGet.CommandLine.2.8.6\tools\NuGet.exe install "FAKE" -OutputDirectory "Packages" -ExcludeVersion
```

Comment:
Next time you need to update FAKE - you will just run this file instead of Googling parameters for invocation.
Other projects (e.g. [NSubstitute](https://github.com/nsubstitute/NSubstitute)) have this file as well.